uv-publish.py

```
@@ -0,0 +1,78 @@
+
+[pypi]
+repository = https://upload.pypi.org/legacy/
+
+[testpypi]
+repository = https://test.pypi.org/legacy/
+"""
+
+
+def main() -> int:
+    print("Building package")
+    subprocess.run(['uv', 'build'])
```

Pylint (3.3.7)

[warning] 'subprocess.run' used without explicitly defining the value for 'check'.

(W1510)